test/e2e/e2e_test.go

```diff
@@ -182,4 +182,33 @@ var _ = Describe("Workload cluster creation", func() {
 			}, result)
 		})
 	})
+
+	Context("Creating a single control-plane cluster", func() {
+		It("Should create a cluster with 1 worker node and can be scaled", func() {
+			By("Initializes with 1 worker node")
+			clusterctl.ApplyClusterTemplateAndWait(ctx, clusterctl.ApplyClusterTemplateAndWaitInput{
+				ClusterProxy: bootstrapClusterProxy,
+				ConfigCluster: clusterctl.ConfigClusterInput{
+					LogFolder:                clusterctlLogFolder,
+					ClusterctlConfigPath:     clusterctlConfigPath,
+					KubeconfigPath:           bootstrapClusterProxy.GetKubeconfigPath(),
+					InfrastructureProvider:   clusterctl.DefaultInfrastructureProvider,
+					Flavor:                   clusterctl.DefaultFlavor,
```

Comment:
The flavour will need to be set to `ci-gke-custom-subnet` so that it picks up your template file.

Reply:
Updated